packages/puppeteer-core/src/bidi/Page.ts

```diff
@@ -537,6 +537,10 @@ export class BidiPage extends Page {
   }
 
   override async setCacheEnabled(enabled?: boolean): Promise<void> {
+    if (!this.#browserContext.browser().cdpSupported) {
+      await this.#frame.browsingContext.setCacheBypass(enabled ?? false);
```

Comment:
```suggestion
      await this.#frame.browsingContext.setCacheBypass(!enabled ?? true);
```

Reply:
did you mean to change to `enabled ?? true`?

Reply:
ah sorry, you are totally write, not sure what I was reading :D